__tests__/app.test.js

```diff
@@ -29,3 +30,17 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api", () => {
+  test("should return a list of endpoints with descriptions,queries and exampleResponse", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        const endpoints = body.endpoints;
+
+        expect(typeof endpoints).toBe("object");
+        expect(endpoints).not.toBeNull();
```

Comment:
We just need an expect statement in here to ensure that the endpoints that are coming back match the json file. So something like `expect(endpoints).toEqual(endpointsData)` should do the trick 🙂